plugins/amoeba/openmmapi/include/openmm/AmoebaGeneralizedKirkwoodForce.h

```diff
@@ -76,25 +76,42 @@ class OPENMM_EXPORT_AMOEBA AmoebaGeneralizedKirkwoodForce : public Force {
      */
     int addParticle(double charge, double radius, double scalingFactor);
 
+    /**
+     * Add the  parameters for a particle.  This should be called once for each particle
+     * in the System.  When it is called for the i'th time, it specifies the parameters for the i'th particle.
```

Comment:
Since there are two different versions of `addParticle()`, maybe document what the difference between them is?  Especially what values get filled in for `descreenRadius` and `neckFactor` when they aren't specified.